campaigns/20150429-sign-letter.html

```diff
@@ -147,6 +153,33 @@
 
 
 <div class="container">
+<!-- <<<<<<< HEAD -->
```

Comment:
This one is up to you so much different content had been added to the page I could not be clear as what was old and what was new.  I tried to remove what was right but leave everything else to add back in at your discretion.